src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Events
+{
+    // Note: Doesn't implement IApplicationReviewEvent as it's an externally published event.
```

Comment:
Should we maybe put this declaration closer to the one job that uses it as it seems very specific maybe.

Reply:
But it is still a domain event isn't it? We're just acting on it as opposed to raising it.

Reply:
👍 I was just thinking because it is used in one place.